bundles/pom.xml

```diff
@@ -5,634 +5,1249 @@
   <modelVersion>4.0.0</modelVersion>
 
   <parent>
+
```

Comment:
Please restore formatting of this file.

Reply:
I did not format this file personally. It was most probably modified by the command
create_openhab_binding_skeleton.sh "ws980wifi" "Joerg Dokupil" "prof2004" or
mvn clean install -pl :org.openhab.binding.ws980wifi

Please advise, what you want me to do in detail.

Reply:
This file with `xx` is added as reference, but you can just delete this file.

Reply:
This seems to be caused by a bug in the skeleton script on windows. Probably easiest to take the original file from openhad~addons repo main  branch, copy it over your  erosion of the file and.add the line with your binding at the right sorted place.

Reply:
have made no modifications, as far as I know.
What do you want that I do in detail?

Reply:
The skeleton script added your binding on line 762, but the script also added a newline on each row (the bug on wi does). Therefor just restore the original version of this file, and add the row of your own binding ack.

Reply:
Please have another look at the differences - still some additional linefeeds are added.